src/mixins/filing-template-mixin.ts

```diff
@@ -558,14 +561,17 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
       if (effectiveDate >= this.getCurrentJsDate) this.setEffectiveDate(effectiveDate)
     }
 
-    // NB: Staff role is mutually exclusive with premium account.
-    if (this.isRoleStaff) {
+    if (status === FilingStatus.CHANGE_REQUESTED) {
+      // if user is editing a (previously filed) filing, waive fees as they don't have to pay again
+      this.parseStaffPayment({ header: { waiveFees: true } } as any)
```

Comment:
This is to zero out the Fee Summary. The actual determination of whether to redirect to Pay or not is in Actions.vue, and it's actually the Legal API that tells us if we have to pay or skip pay.